guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -15,4 +15,12 @@ There are no upgrade warnings with Foreman {ProjectVersion}.
 [id="foreman-deprecations"]
 == Deprecations
 
-There are no deprecations with Foreman {ProjectVersion}.
+=== Running Foreman on {EL} 8 removal in Foreman 3.13
```

Comment:
Is there a method, within the release note, to link users to the guide on upgrading their EL 8 to EL 9?

Reply:
Oh yes, we have such a guide now. Good point.

Reply:
Now with a link to the correct upgrade section.